src/DynamoPackagesUI/ViewModels/PackageManagerViewModel.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Dynamo.DynamoPackagesUI.Utilities;
+using Dynamo.PackageManager;
+using Dynamo.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dynamo.DynamoPackagesUI.ViewModels
+{
+    /// <summary>
+    /// Package Manager View Model
+    /// </summary>
+    public class PackageManagerViewModel
+    {
+        public const string PACKAGE_MANAGER_URL = "http://dynamopackagemanager.com.s3-website-us-east-1.amazonaws.com";
```

Comment:
This is good to start with but we would also want it to be configurable so that we can re-direct to staging site if needed.